src/resolvers/query/workspace-service-account/index.unit.test.js

```diff
@@ -0,0 +1,95 @@
+import resolvers from "resolvers";
+import casual from "casual";
+import { graphql } from "graphql";
+import { makeExecutableSchema } from "graphql-tools";
+import { importSchema } from "graphql-import";
+import { WORKSPACE_ADMIN } from "constants";
+
+// Import our application schema
+const schema = makeExecutableSchema({
+  typeDefs: importSchema("src/schema.graphql"),
+  resolvers
+});
+
+// Define our mutation
+const query = `
+  query workspaceServiceAccount(
+    $serviceAccountUuid: Uuid!
+  ) {
+    workspaceServiceAccount(
+      serviceAccountUuid: $serviceAccountUuid
+    ) {
+      id
+      label
+      apiKey
+      entityType
+      entityId: workspaceUuid
+      category
+      active
+      lastUsedAt
+      createdAt
+      updatedAt
+    }
+  }
+`;
+
+describe("workspaceServiceAccount", () => {
+  test("typical request is successful", async () => {
+    // Create mock user.
+    const user = {
+      id: casual.uuid,
+      username: casual.email,
+      roleBindings: [
+        {
+          role: WORKSPACE_ADMIN,
+          workspace: { id: casual.uuid }
+        }
+      ]
+    };
+
+    // Mock up some db functions.
+    const workspaceServiceAccount = jest.fn();
+
+    // Construct db object for context.
+    const db = {
+      query: { workspaceServiceAccount }
+    };
+
+    const vars = {
+      serviceAccountUuid: casual.uuid
+    };
+
+    // Run the graphql mutation.
+    const res = await graphql(schema, query, null, { db, user }, vars);
+    expect(res.errors).toBeUndefined();
+    expect(workspaceServiceAccount.mock.calls.length).toBe(1);
```

Comment:
I prefer the slightly more "literate" version:

```suggestion
    expect(workspaceServiceAccount).toHaveBeenCalledTimes(1);
```

(or `.toHaveBeenCalled()` if we don't care about how many times.)

Reply:
It may also be worth asserting that we call this mutation with using `expect(workspaceServiceAccount).toHaveBeenCalledWith` -- see https://github.com/astronomer/houston-api/blob/58b30f8c0e16d48e437c5a3fd6a58fee9dcc4a96/src/resolvers/mutation/update-service-account/index.unit.test.js#L74-L83 for an example